core/src/gl/typedMesh.h

```diff
@@ -37,37 +38,45 @@ class TypedMesh : public VboMesh {
      * after _byteOffset in the mesh vertex data memory
      */
     template<class A>
-    void updateAttribute(GLintptr _byteOffset, unsigned int _nVerts, const A& _newAttributeValue) {
+    void updateAttribute(std::array<int,2> _vertexRange, const A& _newAttributeValue, size_t _attribOffset = 0) {
```

Comment:
It's just c++11 version of the c-array + additional API that we don't need here. So I wouldn't mind to change this to standard c arrays.  I thought to rather add an IntRange type in a utils header to make the intended use more clear - but really a plain 2 element array should be ok.